src/pybind/mgr/openattic/module.py

```diff
@@ -24,8 +23,17 @@ def __init__(self, *args, **kwargs):
         super(Module, self).__init__(*args, **kwargs)
 
     def serve(self):
-        cherrypy.config.update({'server.socket_host': '0.0.0.0',
-                                'server.socket_port': 8080,
+        server_addr = self.get_localized_config('server_addr', '::')
+        server_port = self.get_localized_config('server_port', '8080')
+        if server_addr is None:
+            raise RuntimeError(
+                'no server_addr configured; '
+                'try "ceph config-key set mgr/{}/{}/server_addr <ip>"'.format(
```

Comment:
Maybe "set" and "put" do exactly the same, but ceph config-key documentation uses "put" instead of "set", shouldn't we suggest the same as the documentation? http://docs.ceph.com/docs/jewel/man/8/ceph/

Reply:
The docs link you used is for the `jewel` version. In the latest documentation the command is `set`. http://docs.ceph.com/docs/master/man/8/ceph/

Reply:
It's still "put" in the latest version.

Reply:
After scroll the page I saw the usage of "set", maybe it's an inconsistency on the documentation, but both work.